visualization/app/codeCharta/util/customConfigBuilder.ts

```diff
@@ -22,7 +23,8 @@ export function buildCustomConfigFromState(configName: string, state: State): Cu
 			dynamicSettings: undefined,
 			fileSettings: undefined,
 			treeMap: undefined
-		}
+		},
+		camera: cameraPosition
```

Comment:
What's happening if we have stored an old custom config without the camera position property?
I think there could be an error, if we try to resolve the old custom config structure against the new one.

Reply:
Good point. I haven't thought about backwards compatibility. You are right, it will throw an error.

Custom configs are still experimentally. Do you think, we should support both formats? My first thought is to keep it simple and don't transform old format into new one. I wouldn't even increment major version, as experimental features are allowed to introduce breaking changes.

But being backwards compatibly would be a simple check for old field and applying it to new field. So we could easily support it. 

What do you think?

Reply:
Good suggestion. Let's add the check and a TODO to remove it in 3 month. "TODO [2022-10-11] ...".
I think we can ignore such old and deprecated configs without the camera properties from October on.

Reply:
done. But I noticed that we need to adjust metric templates as well. Therefore we would need a major update, once we remove the wrapper.

Reply:
as discussed tested and todo added for later removal